### ISSUE TITLE
Cannot install on arm

### ISSUE THREAD

**Issue by EdwardMoyse** [HSF/phoenix#627]
The failure mode is:
```
➤ YN0009: │ canvas@npm:2.11.2 couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/j6/cv3xx2910nx_pg7v2t8czqzc0000gn/T/xfs-34791f8f/build.log)
➤ YN0009: │ gl@npm:6.0.2 couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/j6/cv3xx2910nx_pg7v2t8czqzc0000gn/T/xfs-d881d6fe/build.log)
➤ YN0007: │ nx@npm:16.6.0 [73c33] must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 1m 37s
```

I have tried:
```
brew install pkg-config cairo pango libpng jpeg giflib librsvg
```

as proposed here: https://stackoverflow.com/questions/74239415/unable-to-install-node-canvas-on-m1-mac

But this didn't work. I think it MIGHT be related to c++17, but the instructions to:
```
# For macOS Ventura you may need to specify C++ version, e.g. yarn install -std=c++17
```
no longer work:

```
phoenix % yarn install -std=c++17
Unknown Syntax Error: Invalid option name ("-std=c++17").
```

**Comment by EdwardMoyse** [comment]
I think this comes from jsroot:
```
└─ jsroot@npm:7.6.0
   └─ canvas@npm:2.11.2 (via npm:^2.11.2)
```

In a fresh directory, I can install just fine with npm:
```
npm install jsroot
emoyse@macminim1umass Temp % ls node_modules/canvas 
CHANGELOG.md	binding.gyp	build		index.js	package.json	types
Readme.md	browser.js	changes		lib		src		util
```

So I'm unsure what the problem is with ARM (btw I also tried on a non-mac arm machine, and had the same issues)

**Comment by EdwardMoyse** [comment]
Tried with: 
```
export CXXFLAGS="-stdlib=libc++"
```
from [here](https://dev.to/steckdev/setting-up-m1m2-mac-for-using-package-node-gyp-1d9e). With this recipe, I can install a native module that depends on node-gyp, without problems) 

But `yarn install` of phoenix fails with the usual error.

I tried also:
```
npm_config_target_arch=arm64 npm_config_target_platform=linux npm_config_target_libc=glibc yarn install
``` 
from [here](https://github.com/yarnpkg/yarn/issues/2221#issuecomment-1168971270) with no luck (same error).
